app/src/main/kotlin/at/bitfire/davdroid/sync/AddressBookSyncer.kt

```diff
@@ -59,68 +65,48 @@ class AddressBookSyncer @Inject constructor(
                 logger.warning("No contacts permission, but address books are selected for synchronization")
             return // Don't sync
```

Comment:
The "Don't sync" `return` doesn't do anything, right? Can we just drop it? Or otherwise maybe return/evaluate a Boolean instead?

Reply:
True, but it should have done so. Now I can see, that for the AddressBookSyncer we don't need the permissions check at all. Acquiring the provider will already fail in the base Syncer's "invoke" (old "onPerformSync") method.